docs/usage/shoot_maintenance.md

```diff
@@ -45,12 +45,27 @@ During the daily maintenance, the Gardener Controller Manager updates the Shoot'
  - There is a higher version available and the Shoot opted-in for automatic version updates.
  - The currently used version is `expired`.
 
-Gardener creates events with the type `MaintenanceDone` on the Shoot describing the action performed during maintenance, including the reason why an update has been triggered.
+The target version for machine image upgrades is controlled by the `updateStrategy` field for the machine image in the CloudProfile. Allowed update strategies are `patch`, `minor` and `major`.
 
-```yaml
-MaintenanceDone  Updated image of worker-pool 'coreos-xy' from 'coreos' version 'xy' to version 'abc'. Reason: AutoUpdate of MachineImage configured.
-MaintenanceDone  Updated Kubernetes version '0.0.1' to version '0.0.5'. This is an increase in the patch level. Reason: AutoUpdate of Kubernetes version configured.
-MaintenanceDone  Updated Kubernetes version '0.0.5' to version '0.1.5'. This is an increase in the minor level. Reason: Kubernetes version expired - force update required.
+Gardener (gardener-controller-manager) populates the `lastMaintenance` field in the Shoot status with the maintenance results.
+
+```text
+Last Maintenance:
+    Description:     Control Plane: Updated Kubernetes version from "1.26.4" to "1.27.1". Reason: Kubernetes version expired - force update required
+    State:           Succeeded
+    Triggered Time:  2023-07-28T09:07:27Z
+```
+
+Additionally, Gardener creates events with the type `MachineImageVersionMaintenance` or `KubernetesVersionMaintenance` on the Shoot describing the action performed during maintenance, including the reason why an update has been triggered.
+
+```text
```

Comment:
Those are events on a resource, which are not in a yaml-structure? I verbatim copied them from a resource, to serve as documentation. So I don't think I should format anything here.

```
LAST SEEN   TYPE      REASON                           OBJECT          MESSAGE
30m         Normal    MachineImageVersionMaintenance   shoot/local     Worker pool "local": Updated image from 'coreos' version 'xy' to version 'abc'. Reason: Automatic update of the machine image version is co
nfigured (image update strategy: major).

30m         Normal    KubernetesVersionMaintenance     shoot/local     Control Plane: Updated Kubernetes version from "1.26.4" to "1.27.1". Reason: Kubernetes version expired - force update required.

15m         Normal    KubernetesVersionMaintenance     shoot/local     Worker pool "local": Updated Kubernetes version '1.26.3' to version '1.27.1'. Reason: Kubernetes version expired - force update required.

```

Otherwise, please make suggestion how to format this to yaml.